app/utils/press-release-snapshot.js

```diff
@@ -63,6 +77,16 @@ export default class PressReleaseSnapshot {
       await publicationEvent.save();
     }
 
+    //deletes any files from the database which are in a deleted state but not actually deleted
+    const attachments = this.attachments;
+    if (attachments.length) {
+      attachments.forEach(attachment => {
+        if (attachment.hasDirtyAttributes) {
+          attachment.destroyRecord();
```

Comment:
`await` toevoegen. Als je hier dan met `.map()` werkt ipv `forEach` kan je de promises in parallel laten lopen en awaiten met `Promise.all`.